nextflow_schema.json

```diff
@@ -11,38 +11,63 @@
             "fa_icon": "fas fa-terminal",
```

Comment:
This is too much to be reviewed without a nice interface - will review once its in `dev` and I can use the launcher on the website to check for content / typos ;-)

Reply:
Ok!